### PR TITLE
Adding some troubleshooting info

### PR DESCRIPTION
**Pull request by yadudoc**
Provides documentaion fix for #59
Adding troubleshooting for network issues with IPP.

### CODE REVIEW

File: docs/faq.rst [hunk 1]
```diff
@@ -185,3 +185,33 @@ Here's an example of running a python2.7 code as a bash application:
...
+
+Parsl hangs
+^^^^^^^^^^^
+
+There are a few common situations in which a Parsl script might hang:
+
+1. Circular Dependency in code
+   If an `app` takes a list as an `input` argument and the future returned
+   is added to that list, it creates a circular dependency that cannot be resolved.
+   This situation is described `here <https://github.com/Parsl/parsl/issues/59>`_ in more detail
```
Comment (annawoodard) [inline docs/faq.rst]: detail -> detail.

File: docs/faq.rst [hunk 2]
```diff
@@ -185,3 +185,33 @@ Here's an example of running a python2.7 code as a bash application:
...
+
+2. Workers requested are unable to contact the Parsl client due to one or
+   more issues listed below :
```
Comment (annawoodard) [inline docs/faq.rst]: below : -> below:

File: docs/faq.rst [hunk 3]
```diff
@@ -185,3 +185,33 @@ Here's an example of running a python2.7 code as a bash application:
...
+
+   * Parsl client does not have a public IP (e.g. laptop on wifi)
```
Comment (annawoodard) [inline docs/faq.rst]: Should we add advice here for this case?

Comment (yadudoc) [reply docs/faq.rst]: What options do we have besides getting a publicIp (plugging into a network that does publicIPs) or ssh'ing to a machine that has a publicIp ?

Comment (annawoodard) [reply docs/faq.rst]: Exactly, let's add that.

**Pull request merged**